src/app/types/request-info-form.endpoint.ts

```diff
@@ -31,3 +31,13 @@ export interface Scope {
   description: string
   longDescription: string
 }
+
+enum Errors {
```

Comment:
for this purpose, a type is required not an enum. 

Sinde the error itself will not be a enum

Reply:
Yes, you are right. I will fix it